### PR TITLE
done with list components

### PR DESCRIPTION
**Pull request by Varun-28**
Done with list component. It consists of :

- ordered list (normal)
- unordered list (normal)
- notification list (stacked)

normal-list
stacked-list

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
Clean code with the use of CSS variables and rem units.
Good to merge

**Review by Ronak-Mutha — APPROVED** [review]
Good use of variables and responsive units. Good to merge

File: css/component_styles/list.css [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+    width: 100%;
+    padding: 1rem;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+/* normal list */
+
+.list ol, .list ul{
+    padding-left: 40px;
```
Comment (MaruthiPothuganti) [inline css/component_styles/list.css:16]: This isnt required as the `<ul>` and `<ol>` will have a default value of _padding-left :40px_
image


Unless you had reset in the reset css. You can remove this.

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a change. Everything else looks good to merge.

File: css/component_styles/list.css (hunk 1 above)
Comment (Varun-28) [reply css/component_styles/list.css:16]: yes I have reset it that's why

**Pull request merged**